tutorials/katacoda/4-eventing-intro-broker/step2.md

```diff
@@ -0,0 +1,111 @@
+## Use Case
+The broker and trigger model is useful for complex event delivery topologies like N:M:Z, i.e. there are a multitude of Sources sending events, Functions
+consuming/transforming which are then processed by even more functions and so on. It can get a bit unwieldy to keep track of which Channel is having which events. Also, sometimes you might only want to
+consume only some types of event. You would have to receive all the events, and throw out the ones you’re not interested in. To make these kinds of interactions easier and allow the
```

Comment:
```suggestion
consuming/transforming which are then processed by even more functions and so on. It can get a bit unwieldy to keep track of which Channel is having which events. Also, sometimes you might only want to
consume specific types of events. You would have to receive all the events, and throw out the ones you’re not interested in. To make these kinds of interactions easier and allow the
```